spec/controllers/anonymous_feedback/export_requests_controller_spec.rb

```diff
@@ -18,34 +18,33 @@
 
       it "sets the flash" do
         do_request
-
         expect(flash[:notice]).to include "foo.bar@example.gov.uk"
       end
     end
 
-    context "with a path" do
+    context "with a `paths` parameter" do
       let!(:stub_request) do
         stub_support_api_feedback_export_request_creation(notification_email: "foo.bar@example.gov.uk",
-                                                      path_prefix: "/foo",
+                                                      path_prefixes: ["/foo"],
                                                       from: "2015-05-01",
                                                       to: "2015-06-01",
                                                       organisation: nil)
       end
 
-      let(:do_request) { post :create, params: { path: "/foo", from: "2015-05-01", to: "2015-06-01" } }
+      let(:do_request) { post :create, params: { paths: ["/foo"], from: "2015-05-01", to: "2015-06-01" } }
```

Comment:
In the views we set a `paths` hidden field set to the value of `ScopeFilterPresenter#paths` which is a comma separated string, but here we're testing using a `paths` param that is an array of strings.  I've just spotted the `.instance_of?(String)` in the controller itself and I think it's only there to get around this test.  If we use a string here then we can drop that type check.

Reply:
I've added two extra tests to illustrate that the `paths` param can be 
- an array of values: ["/foo", "/bar", "/baz] (test already existed)
- a string with a comma separated list of values: "/foo, /bar, /baz" (new test)
- a string with a single value: "/foo" (new test)

Because `clean_paths` runs before the params.permit method in the controller, it will take all these three possible scenarios and always return `paths` as an array of values.

Reply:
:+1: